### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by IlyaRucavitcyn**
- [x] The Gulp deploy task is set

### CODE REVIEW

File: src/hello.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+fgfherthutryhjetyh
```
Comment (shemerey) [inline src/hello.txt]: what is that ?

File: gulpfile.js [hunk 2]
```diff
@@ -46,15 +48,15 @@ gulp.task('concatjs', function() {
...
 		.pipe(cssmin())
-		.pipe(gulp.dest('css'));
+		.pipe(gulp.dest('css/*.css'));
 });
 
 gulp.task('jsmin', function() {
   return gulp.src('js/*.js')
     .pipe(uglify())
-    .pipe(gulp.dest('js'));
+    .pipe(gulp.dest('../vanga.by_production/js/'));
```
Comment (shemerey) [inline gulpfile.js]: what it the dir `vanga.by_production`  ?

File: gulpfile.js [hunk 3]
```diff
@@ -46,15 +48,15 @@ gulp.task('concatjs', function() {
 });
 
 gulp.task('cssmin', function () {
-	gulp.src('css/*.css')
+	gulp.src('../vanga.by_production/css/*.css')
```
Comment (shemerey) [inline gulpfile.js]: what it the dir `vanga.by_production`  ?

**Comment by IlyaRucavitcyn** [comment]
Sorry for all of mentioned above, tried to look trough again and corrected

**Pull request merged**